cloudbuild.yaml

```diff
@@ -78,8 +82,7 @@ steps:
       - TASKCLUSTER_ACCESS_TOKEN
 timeout: 1800s
 images:
-  - ${_IMAGE_NAME}:${_IMAGE_VERSION}
-  - ${_DEPLOY_IMAGE_NAME}:${_IMAGE_VERSION}
+  - ${_DEPLOY_IMAGE_NAME}:latest
```

Comment:
we only allow gcloud build to push this deploy image automatically, as we are pushing main monoimage manually already